libcodechecker/analyze/analyzers/result_handler_plist_to_stdout.py

```diff
@@ -91,10 +92,11 @@ def __print_bugs(self, reports, files):
                 LOG.debug(report + ' is skipped (in ' + f_path + ")")
                 continue
 
-            if self.suppress_handler and \
-                    self.suppress_handler.get_suppressed(report):
-                LOG.debug(report + " is suppressed by suppress file.")
-                continue
+            # TODO: Uncomment this once #662 is merged.
```

Comment:
@gyorb Please put a `[WIP]` on this request and **do not merge** until #662 is merged, as this is only commented out so tests relevant to suppression generation can run.